src/Security/Group.php

```diff
@@ -377,6 +377,38 @@ public function collateAncestorIDs()
     }
 
     /**
+     * @param string|int|Group $group
+     * @return bool
+     */
+    public function inGroup($group) {
```

Comment:
Would be good if the phpdoc told me what this method did. :)

Tests would also be nice.